backend/tests/v5/e2e/routes/teamspaces/projects/containers/containers.test.js

```diff
@@ -157,13 +167,14 @@ const testGetContainerStats = () => {
 			expect(res.body.code).toEqual(templates.containerNotFound.code);
 		});
 
-		test('should fail if the container doesn\'t exist', async () => {
-			const res = await agent.get(`${route('jibberish')}?key=${users.tsAdmin.apiKey}`).expect(templates.containerNotFound.status);
-			expect(res.body.code).toEqual(templates.containerNotFound.code);
+		test('should fail if the container does not exist', async () => {
+			const res = await agent.get(`${route('jibberish')}?key=${users.tsAdmin.apiKey}`).expect(templates.modelNotFound.status);
```

Comment:
could we keep it as `containerNotFound`?

You can catch the `modelNotFound` error and check the req.params to determine whether we need containerNotFound or federationNotFound